test/regress/regress-38.txt

```diff
@@ -0,0 +1,11 @@
+;;; TOOL: wat2wasm
```

Comment:
What is 38?  I though these used bug number?

Reply:
Most of the regression tests seemed to be just numbered sequentially, so I went with that since we didn't have a bug number for this bug exactly. Can easily renumber it this this PR (done).